pkg/frontend/admin_vm_actions.go

```diff
@@ -11,43 +11,33 @@ import (
 
 	"github.com/Azure/ARO-RP/pkg/api"
 	"github.com/Azure/ARO-RP/pkg/database/cosmosdb"
+	"github.com/Azure/ARO-RP/pkg/frontend/adminactions"
 )
 
-type azVmActionsWrapper struct {
-	vmName string
-	doc    *api.OpenShiftClusterDocument
-}
-
-func (f *frontend) newAzureActionsWrapper(log *logrus.Entry, ctx context.Context, vmName, resourceID string, vars map[string]string) (azVmActionsWrapper, error) {
-	err := validateAdminVMName(vmName)
-	if err != nil {
-		return azVmActionsWrapper{}, err
-	}
+func (f *frontend) prepareAdminActions(log *logrus.Entry, ctx context.Context, vmName, resourceID string, vars map[string]string) (azureActions adminactions.AzureActions, doc *api.OpenShiftClusterDocument, err error) {
```

Comment:
I'd prefer we didn't mix return types here.. Can we please swap this back to not use naked returns?